Sources/Sentry/SentryTracer.m

```diff
@@ -395,6 +397,106 @@ - (void)removeTagForKey:(NSString *)key
     }
 }
 
+- (void)setMeasurement:(NSString *)name value:(NSNumber *)value
+{
+    _measurements[name] = @{ @"value" : value, @"unit" : @"none" };
```

Comment:
The agreement is that, if not provided, `unit` is ommited and the server infers the `none` value.